changelogs/fragments/allow-similar-name-per-zone.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - Implement distinquish per zone to allow identical naming of the same resource type per zone (https://github.com/cloudscale-ch/ansible-collection-cloudscale/pull/46).
```

Comment:
If you wanted to write it as you proposed, you should probably add quotes as it doesn't seem grammatically correct otherwise:

> Implement "distinguish per zone" to allow identical naming of the same resource type per zone.

Also note it's "distinguish" with a "g", not a "q".

Anyway, I would prefer something a bit more descriptive:

> Enable the reuse of network and server group names, as long as they are not in the same zone (https://github.com/cloudscale-ch/ansible-collection-cloudscale/pull/46).